task160_FinalTask/src/test/java/automationpractice/com/pages/CreateAnAccountPage.java

```diff
@@ -0,0 +1,95 @@
+package automationpractice.com.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateAnAccountPage extends BasePage {
+    @FindBy(id = "submitAccount")
+    private WebElement REGISTER_BUTTON;
+    //YOUR PERSONAL INFORMATION
+    @FindBy(id = "customer_firstname")
+    private WebElement FIRST_NAME_INPUT;
+    @FindBy(id = "customer_lastname")
+    private WebElement LAST_NAME_INPUT;
+    @FindBy(id = "email")
+    private WebElement EMAIL_INPUT;
+    @FindBy(id = "passwd")
+    private WebElement PASSWORD_INPUT;
+    @FindBy(id = "days")
+    private WebElement DAY_OF_BIRTH_SELECT;
+    @FindBy(id = "months")
+    private WebElement MONTH_OF_BIRTH_SELECT;
+    @FindBy(id = "years")
+    private WebElement YEAR_OF_BIRTH_SELECT;
+    @FindBy(id = "newsletter")
+    private WebElement NEWSLETTER_CHECKBOX;
+    @FindBy(id = "optin")
+    private WebElement SPECIALOFFER_CHECKBOX;
+    //YOUR ADDRESS
+    @FindBy(id = "firstname")
+    private WebElement ADDRESS_FIRST_NAME_INPUT;
+    @FindBy(id = "lastname")
+    private WebElement ADDRESS_LAST_NAME_INPUT;
+    @FindBy(id = "company")
+    private WebElement COMPANY_INPUT;
+    @FindBy(id = "address1")
+    private WebElement ADDRESS_INPUT;
+    @FindBy(id = "city")
+    private WebElement CITY_INPUT;
+    @FindBy(id = "id_state")
+    private WebElement STATE_SELECT;
+    @FindBy(id = "postcode")
+    private WebElement POSTCODE_INPUT;
+    @FindBy(id = "id_country")
+    private WebElement COUNTRY_SELECT;
+    @FindBy(id = "other")
+    private WebElement ADDITIONAL_INFORMATION_INPUT;
+    @FindBy(id = "phone")
+    private WebElement HOME_PHONE_INPUT;
+    @FindBy(id = "phone_mobile")
+    private WebElement MOBILE_PHONE_INPUT;
+    @FindBy(id = "alias")
+    private WebElement ADDRESS_ALIAS_INPUT;
+
+    public CreateAnAccountPage() {
+        REGISTER_BUTTON.isDisplayed();
+    }
+
+    public LoggedHomePage register(String firstName,
```

Comment:
If method has too much input parameters, then it is hard to read and use this method. I highly recommend to create model class (f.e. UserModel/User/etc.) and use it as parameter

Reply:
Default values are added in the UserAccountModel class.

CreateAnAccountPage "register" method modified just to user default UserAccountMode and typeIn/select default data from model into fields on UI.

CreateAnAccountTest modified and just use UserAccountModel object with default values.